control-plane/pkg/reconciler/trigger/trigger.go

```diff
@@ -142,7 +142,9 @@ func (r *Reconciler) reconcileKind(ctx context.Context, trigger *eventing.Trigge
 	if err != nil {
 		return statusConditionManager.failedToResolveTriggerConfig(err)
 	}
-	statusConditionManager.subscriberResolved()
+	statusConditionManager.subscriberResolved(
+		fmt.Sprintf("Subscriber will receive events with the delivery order: %s", triggerConfig.DeliveryOrder.String()),
```

Comment:
A bit more concise?
```suggestion
		fmt.Sprintf("Delivery order: %s", triggerConfig.DeliveryOrder.String()),
```

Reply:
I prefer this wording, because the name of the reason is still `SubscriberResolved`...

Reply:
Yeah, make sense, thanks!